lib/restbase.js

```diff
@@ -307,6 +318,17 @@ RESTBase.prototype._request = function (req) {
             self.metrics.endTiming([statName + statusClass, statName + 'ALL'], startTime);
             throw err;
         });
+        if (match.permissions
```

Comment:
@gwicke 
a) we don't really need to apply them to nested requests, because they are set on API level, so we will always check it before (or in parallel with) doing any internal request. I've made it like this specially to avoid adding extra logic to verify if we should or should not check permissions.
b) right now the cookie headers are always passed to internal requests